### PR TITLE
Fix english

### PR DESCRIPTION
**Pull request by pabedu**

### CODE REVIEW

File: Proposal.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Hi internals@,
 
-PHP support object oriented programming (aka OOP), and one of the base of OOP is polymorphism.
-So it’s currently possible to use polymorphism in PHP code.
-However, in some case, using it can be a PITA, because PHP does not implements natively the Null Object Pattern.
-This is a (very) simple use case to illustrate that:
+PHP supports Object Oriented Programming (aka OOP), and one of the ground rule of OOP is polymorphism.
+Polymorphism can currently be used in your PHP code, however, in some cases, using it can be a PITA because PHP does not natively implement the Null Object Pattern.
```
Comment (CornuotKevin) [inline Proposal.md]: , however, in some cases,
however in some cases mieux non ?

File: Proposal.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ class parser
 }
 ```
 
-Now, imagine that you don’t want to log any error.
+Now, imagine you don’t want to log any error.
```
Comment (CornuotKevin) [inline Proposal.md]: Now, imagine
Now imagine là par contre c'est mieux :)

File: Proposal.md [hunk 3]
```diff
@@ -83,23 +83,22 @@ class parser
...
 
-It’s a very better solution than the previous one because code complexity is not increased by `if` statement, and code readability is improved.
-But now imagine that you use it in lot of classes: you must define lot of `blackhole*` classes, and doing that can be rapidly a PITA!
-So, i think that it must be interesting to implements natively in the language the null object pattern as a class which has the following behavior:
-
-1) an instance can replace any interface or class in term of type hinting (so the class implements or extends virtually all classes and interfaces);
-2) all call to a method upon an instance return `$this` to avoiding the special case of returning "no object ».
-3) an instance do absolutely nothing!
+It’s a much better solution than the previous one because code complexity is not increased by this `if` statement, and code readability is improved.
+But now imagine you use it in lots of classes: you must define lots of `blackhole*` classes, and doing that can be rapidly a PITA!
```
Comment (CornuotKevin) [inline Proposal.md]: But now imagine you use it
Imagine using it

File: Proposal.md [hunk 4]
```diff
@@ -83,23 +83,22 @@ class parser
...
+So, we think it would be interesting to natively implement the Null Object Pattern in PHP as a class which would have the following behaviors:
```
Comment (CornuotKevin) [inline Proposal.md]: So, we think
Therefore we think

File: Proposal.md [hunk 5]
```diff
@@ -83,23 +83,22 @@ class parser
...
 
-Its name can be:
+1. an instance can replace any interface or class in term of type hinting (so the class virtually implements or extends all classes and interfaces)
+2. every call to a method upon an instance return `$this` to avoid the special case of returning "no object".
+3. an instance does absolutely nothing!
 
+Its name could be:
```
Comment (CornuotKevin) [inline Proposal.md]: J'aurai pas mis le Its même si grammaticalement correct, mais plutôt un truc du genre You can name it, etc..

Comment (mageekguy) [inline Proposal.md]: Names for class could be ?

Comment (CornuotKevin) [inline Proposal.md]: The name of the class could be dans ce cas là

Comment (mageekguy) [inline Proposal.md]: @pabedu any feedback about "The name of the class could be:"?

Comment (pabedu) [inline Proposal.md]: I'd go with "You could name this class:" !

**Comment by CornuotKevin** [comment]
:+1:

**Pull request merged**